### PR TITLE
MAINT-26842: UI glitch in permission field

### PR DESCRIPTION
**Pull request by AzmiTouil**
hide the suggester list of permissions field when scrolling in the app center drawer

### CODE REVIEW

File: app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/drawer/ExoAppCenterDrawer.vue [hunk 1]
```diff
@@ -286,6 +286,12 @@ export default {
       this.permissions.push(...groups);
     },
   },
+  mounted () {
+    $('.formContent').on( 'scroll', function(){
```
Comment (ahamdi) [inline app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/drawer/ExoAppCenterDrawer.vue:290]: Couldn't you manage this by CSS ?

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/drawer/ExoAppCenterDrawer.vue (hunk 1 above)
Comment (AzmiTouil) [reply app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/drawer/ExoAppCenterDrawer.vue:290]: No, I could not

Comment (ahamdi) [reply app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/drawer/ExoAppCenterDrawer.vue:290]: Ok thanks

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**